database/migrations/2022_11_06_111641_add_training_list_enforce_hour_eligibility_column.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('training_waiting_list', function (Blueprint $table) {
+            $table->enum('enforce_hour_requirement', ['yes', 'no'])->after('flags_check')->defalt('yes');
```

Comment:
Why use an enum here when it is a true/false value?

Reply:
This mainly stems from personal preference as opposed to any benefit in this specific situation. 

My way of thinking is generally if we use 1/0 for this column, and in the future there is a case for expanding this to something along the lines of:

Enforces Hour Requirement
- yes
- reduced
- no

Then there are minimal database updates required. The above may not be the best example, and I wouldn't argue against it being a Boolean for this column, but the reasoning is to provide some flexibility / scalability